docs/core/diagnostics/distributed-tracing.md

```diff
@@ -114,13 +114,14 @@ The `ActivityListener` class provides a different callbacks to handle different
 
 ```csharp
 
-ActivityListener listener = new ActivityListener()
-
-ShouldListenTo = (activitySource) => object.ReferenceEquals(source, activitySource),
-ActivityStarted = activity => /* Handle the Activity start event here */ DoSomething(),
-ActivityStopped = activity => /* Handle the Activity stop event here */ DoSomething(),
-SampleUsingParentId = (ref ActivityCreationOptions<string> activityOptions) => ActivitySamplingResult.AllData,
-Sample = (ref ActivityCreationOptions<ActivityContext> activityOptions) => ActivitySamplingResult.AllData
+ActivityListener listener = new ActivityListener
```

Comment:
```suggestion
var listener = new ActivityListener
```

Reply:
@Youssef1313, if it was my code, that's how it would be written.

What do you think, @BillWagner?

Reply:
I would use `var` in that instance, but I'm not going to hold up merging for that reason alone.